plugins/modules/ec2_vpc_net.py

```diff
@@ -194,22 +194,23 @@
           sample: pk_vpc4
 '''
 
+from time import sleep, time
```

Comment:
nit: all of the other imports have been split out over 1 line per-import, let's be consistent.
```suggestion
from time import sleep
from time import time
```